docs/DeveloperGuide.md

```diff
@@ -503,13 +503,68 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **Extensions**
 
 * 2a. User uses list food reviews command with the wrong syntax.
+
     * 2a1. FoodWhere sends an error message to the User, indicating that the syntax is incorrect, and attaches the correct syntax format in the message.
 
       Use case ends.
 
 ****
 
-**Use case 7: Exiting the program**
+**Use case 7: Edit a food stall**
+
+**MSS**
+
+1. User starts FoodWhere.
+2. User enters the command to list food stalls.
+3. FoodWhere displays all food stalls.
+4. User enters the command to edit a particular food stall.
+5. FoodWhere edits the food stall and confirms with a success message that the food stall has been edited.
+
+   Use case ends.
+
+**Extensions**
+
+* 4a. User issues the edit food stall command with the wrong syntax
+
+    * 4a1. FoodWhere sends an error message to the User, indicating that the syntax is incorrect, and attaches the correct syntax format in the message.
+
+      Use case ends.
+  
+* 4b. FoodWhere detects an error in the entered data. (Invalid index)
+    * 4b1.FoodWhere sends an error message to the User, indicating the index used for the edit food stall command is incorrect, and attaches the correct syntax format in the message.
```

Comment:
I would think 4a and 4b can be combined, and instead of "wrong syntax" we can change it to "User issues the edit food stall command with invalid data", the particular extension here seems similar enough that we can drop it, making the DG clearer.

Reply:
Updated as per suggestion